examples/example_thymine.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Designing a synthesis route and executing it on a robot\n",
+    "In this example we show how to use the endpoints of the `rxn4chemistry` package. \n",
+    "- We start from a target molecule represented in a smiles notation.\n",
+    "- We then use the models to predict a retrosynthetic path.\n",
+    "- Once we have chosen the desired path, we start a new synthesis on the robotic hardware.\n",
+    "- We monitor the progress of the execution and finally download a pdf report from the analysis.\n",
+    "\n",
+    "## Instantiating the wrapper\n",
+    "Setup the wrapper using a valida API key. You can get one on the IBM RXN website from [here](https://rxn.res.ibm.com/rxn/user/profile)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from rxn4chemistry import RXN4ChemistryWrapper\n",
+    "api_key = 'API_KEY'\n",
+    "rxn4chemistry_wrapper = RXN4ChemistryWrapper(api_key=api_key)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a project\n",
+    "\n",
+    "Create a project, you can easily check the identifier associated to it in the response."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = rxn4chemistry_wrapper.create_project('rxn4chemistry_examples')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can get your project id from the wrapper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rxn4chemistry_wrapper.project_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run a retrosynthesis prediction using the wrapper\n",
+    "Running a retrosynthesis is as easy as picking a molecule and calling a one-liner.\n",
+    "\n",
+    "In this example, we will be predicting a path for thymine."
```

Comment:
@asobczyk ping antonio to ask if the molecule, and especially the sequence of actions make sense.